ibm/resource_ibm_cis_origin_pool_test.go

```diff
@@ -86,8 +91,6 @@ func TestAccIBMCisPool_FullySpecified(t *testing.T) {
 }
 
 func TestAccIBMCisPool_CreateAfterManualDestroy(t *testing.T) {
-	//t.Parallel()
-	t.Skip()
```

Comment:
few test cases are intentionally skipped.. can you please revert back this change..

Reply:
done